### PR TITLE
New header created

### PR DESCRIPTION
**Pull request by ElsonOtake**
A blue-violet header was created. Let me know if the color is out of our standard.

### CODE REVIEW

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @ElsonOtake , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

[✔️](https://emojipedia.org/check-mark/) Clean Pull Request and Professional README
[✔️](https://emojipedia.org/check-mark/) Great job on this Hello World!

## Required Changes ♻️
_Check the comments under the review._

- [x] To install the linters properly, try to follow [this repo's readme](https://github.com/microverseinc/linters-config/tree/master/html-css). For the links to be checked on GitHub, they must be inside a .github folder like the documentation indicates.
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ElsonOtake** [comment]
> Hi @ElsonOtake ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Highlights
> [✔️](https://emojipedia.org/check-mark/) Clean Pull Request and Professional README [✔️](https://emojipedia.org/check-mark/) Great job on this Hello World!
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> * [x]  To install the linters properly, try to follow [this repo's readme](https://github.com/microverseinc/linters-config/tree/master/html-css). For the links to be checked on GitHub, they must be inside a .github folder like the documentation indicates.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi Julian, sorry, but I didn't understand what is the problem. The file was created in the folder "workflows" that is inside of the folder ".github". Is it correct? I don't know if I made the creation correctly, but I used the line commands to create the file. The "mkdir" to create the folders and the "nano" editor to fill in the file "linters.yml". Is the issue in this way of creating the file?

Thanks in advance,

Elson

**Comment by ElsonOtake** [comment]
> > Hi @ElsonOtake ,
> > Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> > ## Highlights
> > [✔️](https://emojipedia.org/check-mark/) Clean Pull Request and Professional README [✔️](https://emojipedia.org/check-mark/) Great job on this Hello World!
> > ## Required Changes ♻️
> > _Check the comments under the review._
> > 
> > * [x]   To install the linters properly, try to follow [this repo's readme](https://github.com/microverseinc/linters-config/tree/master/html-css). For the links to be checked on GitHub, they must be inside a .github folder like the documentation indicates.
> > 
> > Cheers and Happy coding!👏👏👏
> > Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> > **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> > _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
> 
> Hi Julian, sorry, but I didn't understand what is the problem. The file was created in the folder "workflows" that is inside of the folder ".github". Is it correct? I don't know if I made the creation correctly, but I used the line commands to create the file. The "mkdir" to create the folders and the "nano" editor to fill in the file "linters.yml". Is the issue in this way of creating the file?
> 
> Thanks in advance,
> 
> Elson

Sorry, I forgot to tag @JuliCarracedo.

Elson

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# ignore the test.md file
+test.md
```
Comment (mahbubul14) [inline .gitignore:2]: - [ ] Please add `node_modules` in the `.gitignore` file. As your previous reviewer didn't request this change, I kept it optional this time. But this is actually a mandatory requirement and I think you should follow this for your next projects also.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @ElsonOtake   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Used proper git-flow.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions :arrow_down_small: 

- [ ] Please check the comment below this review. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: .gitignore (hunk 1 above)
Comment (ElsonOtake) [reply .gitignore:2]: Thank you for the advice @mahbubul14. I certainly will use this valuable information.

**Pull request merged**